src/llm/llm_calculator.proto

```diff
@@ -78,4 +78,6 @@ message LLMCalculatorOptions {
     optional bool draft_dynamic_split_fuse = 17;
 
     optional PipelineType pipeline_type = 18 [default = AUTO];
+    
+    optional string allowed_local_media_path = 19;

```

Comment:
Shouldn't it be more deployment-wide setting on the server level? 
It injects frontend option to servable configuration that holds backend settings.